icons/cloudflare/cloudflare-original.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path fill="#F38020" d="M86.496 83.018c-1.163-1.708-3.126-2.693-5.486-2.805l-44.74-.608a.816.816 0 0 1-.689-.38 1.081 1.081 0 0 1-.109-.832 1.233 1.233 0 0 1 1.054-.835l45.136-.608c5.343-.265 11.159-4.779 13.192-10.315l2.581-7.014c.114-.29.139-.606.072-.91-2.905-13.728-14.644-23.968-28.671-23.968-12.936 0-23.915 8.722-27.839 20.819a12.95 12.95 0 0 0-9.267-2.693c-6.216.645-11.191 5.842-11.811 12.325-.15 1.612-.04 3.239.327 4.816C10.105 70.312 2 78.959 2 89.617c0 .948.075 1.895.184 2.843.072.455.436.798.873.798h82.567c.471 0 .907-.343 1.054-.835l.617-2.313c.763-2.731.473-5.238-.799-7.092z"/><path fill="#FAAE40" d="M101.54 60.178c-.399 0-.835 0-1.234.037-.29 0-.546.228-.655.53l-1.746 6.334c-.761 2.731-.471 5.234.801 7.091 1.163 1.708 3.126 2.693 5.486 2.805l9.523.608c.29 0 .545.153.692.38.144.252.183.551.109.832-.156.463-.57.791-1.057.835l-9.919.608c-5.38.265-11.159 4.775-13.192 10.315l-.729 1.933c-.143.38.109.757.511.757h34.086a.902.902 0 0 0 .873-.683c.6-2.251.906-4.571.91-6.901.001-14.028-10.974-25.481-24.459-25.481"/></svg>
```

Comment:
Looks good, but it's still missing the white color inside the logo.
You can use the logo I sent above as a reference

Reply:
Same as the other answer, I won't put the white shape in the center otherwise the glyph of the font will be a solid color .

Reply:
@BenSouchet Aha, so that's why. 

Well in that case you can just make two new icons. `cloudflare-plain` and `cloudflare-plain-watermark`. Make sure those icons are monocolored, and preferably have only a single path. See [SVG Versions](https://github.com/devicons/devicon/wiki/SVG-Versions) and [SVG Standards](https://github.com/devicons/devicon/wiki/SVG-Standards) for reference